packages/nimble-components/src/anchor-menu-item/tests/anchor-menu-item.react.tsx

```diff
@@ -0,0 +1,9 @@
+/* eslint-disable import/no-extraneous-dependencies */
+/* eslint-disable @typescript-eslint/naming-convention */
+import * as React from 'react';
+import { provideReactWrapper } from '@microsoft/fast-react-wrapper';
+import { AnchorMenuItem } from '..';
+
+const { wrap } = provideReactWrapper(React);
+
+export const NimbleButton = wrap(AnchorMenuItem);
```

Comment:
I'm guessing every component shouldn't be exported as `NimbleButton`?

I'm debating between `NimbleAnchorMenuItem` and something with React / Storybook in its name like `NimbleStorybookAnchorMenuItem` or `AnchorMenuItemReactWrapper`. My gut is that we probably want to avoid any chance of confusing these for the web components so it's worth including something like that, but I don't currently have a strong opinion.

Reply:
I went with `NimbleStorybook` so that it's clear within the MDX files that it's a Nimble component and to set expectations that these are not general-purpose React components - 👍

Reply:
Changed it back to `NimbleButton` per https://github.com/ni/nimble/pull/1637#discussion_r1418248363